cmd/alerts/silence/silence_org.go

```diff
@@ -0,0 +1,105 @@
+package silence
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/openshift/osdctl/cmd/common"
+	orgutils "github.com/openshift/osdctl/cmd/org"
+	ocmutils "github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+type AddOrgSilenceCmd struct {
+	organization string
+	alertID      []string
+	duration     string
+	comment      string
+	all          bool
+}
+
+func NewCmdAddOrgSilence() *cobra.Command {
+	AddOrgSilenceCmd := &AddOrgSilenceCmd{}
+	cmd := &cobra.Command{
+		Use:               "org <org-id> [--all --duration --comment | --alertname --duration --comment]",
+		Short:             "Add new silence for alert for org",
+		Long:              `add new silence for specfic or all alerts with comment and duration of alert for an organization`,
+		Args:              cobra.ExactArgs(1),
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			AddOrgSilenceCmd.organization = args[0]
+			AddOrgSilence(AddOrgSilenceCmd)
+		},
+	}
+
+	cmd.Flags().StringSliceVar(&AddOrgSilenceCmd.alertID, "alertname", []string{}, "alertname (comma-separated)")
+	cmd.Flags().StringVarP(&AddOrgSilenceCmd.comment, "comment", "c", "", "add comment about silence")
+	cmd.Flags().StringVarP(&AddOrgSilenceCmd.duration, "duration", "d", "15d", "add duration for silence") //default duration set to 15 days
+	cmd.Flags().BoolVarP(&AddOrgSilenceCmd.all, "all", "a", false, "add silences for all alert")
+
+	return cmd
+}
+
+// Add alert silences to organization's clusters
```

Comment:
stylistic nitpick - Go function docs usually start with the name of the function
```suggestion
// AddOrgSilence Add alert silences to organization's clusters
```